js/main.js

```diff
@@ -1,6 +1,18 @@
 var $cardList = document.querySelector('.card-list');
+var $genres = document.querySelector('.genres');
 
-renderPage1Movies();
+renderPopularMovies1();
+
+$genres.addEventListener('click', function (e) {
+  var highlightBtn = document.querySelector('.highlight');
+  highlightBtn.className = '';
+  e.target.className = 'highlight';
+  removeAllChildren($cardList);
+  if (e.target.outerText === 'Popular') {
+    renderPopularMovies1();
```

Comment:
You should `return` in this conditional so you don't also end up calling `renderMovieList`